mlir/include/mlir/Dialect/ArmSME/Utils/Utils.h

```diff
@@ -42,6 +49,16 @@ std::optional<ArmSMETileType> getSMETileType(VectorType);
 /// Verifies the tile ID (if set) on this tile operation is valid.
 LogicalResult verifyOperationHasValidTileId(Operation *);
 
+using LoopBodyBuilder =
```

Comment:
`LoopBodyBuilder` is quite vague, it does not mention tiles, or that this is a function. 

Maybe just remove this alias and use `auto` rather than `arm_sme::LoopBodyBuilder` at call sites.